implementations/rust/ockam/ockam_command/src/test_utils/node.rs

```diff
@@ -0,0 +1,61 @@
+use assert_cmd::prelude::*;
+use duct::cmd;
+use lazy_static::lazy_static;
+use rand::random;
+use std::cell::RefCell;
+use std::sync::{Arc, Mutex, MutexGuard};
+
+lazy_static! {
+    static ref NODES_POOL: Arc<Mutex<Vec<TestNode>>> = init_node_pool();
+}
+
+fn init_node_pool() -> Arc<Mutex<Vec<TestNode>>> {
+    let expr = cmd!(super::ockam_bin(), "node", "delete", "-a")
+        .env("OCKAM_PROJECT_PATH", "/tmp/ockam")
+        .run()
+        .unwrap();
+    expr.assert().success();
+
+    let mut node_names = vec![];
+    for _ in 0..3 {
+        let node_name = hex::encode(&random::<[u8; 4]>());
+        node_names.push(TestNode::new(node_name));
+    }
+    Arc::new(Mutex::new(node_names))
+}
+
+pub(crate) fn test_nodes() -> MutexGuard<'static, Vec<TestNode>> {
+    NODES_POOL.lock().unwrap()
+}
+
+pub(crate) struct TestNode {
+    name: String,
+    init: RefCell<bool>,
```

Comment:
Since `TestNode`'s are behind a mutex, is better to use `Cell`over `AtomicBool`, right? In any case, I might be able to remove this altogether and just use a plain boolean.